pages/learn/manage/organizations.md

```diff
@@ -0,0 +1,100 @@
+---
+title: Organization management
+---
+
+# Organization management
+
+An **organization** in balenaCloud is a set of users (members and roles), teams, and applications. Which in turn make collaboration easier and access control more streamlined than ever for your applications and devices. Organizations are available for use in all balenaCloud accounts, blending in with existing features and workflows all the while empowering fleet owners to build bigger, better managed fleets.
+
+## What can you do with organizations?
+
+With organizations you can:
+
+* Invite users to an orgnization as members.
+* Create teams of those members for easier collaboration across applications.
+* Bulk provide explicit access to applications to individual members or teams now. 
+* Manage each member, team, or application with easier controls to help meet security and regulatory rules.
```

Comment:
_"Manage X with easier controls"_ is not exactly what helps meet security and regulatory rules... What helps meet security and regulatory rules is the ability set access control to apps at a granular level, with user roles like observer / operator / developer, and the ability to change and revoke such roles. I think...